assets/git.ignore/.vscode/git.ignore/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
delete the entire `git.ignore` folder as it is not relevant and I can see what you were trying to do :)